<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDatasetConverterClass</class>
 <widget class="QMainWindow" name="QtDatasetConverterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDatasetConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1. Open dataset</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Select dataset root directory then click &quot;Open dataset&quot;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="directoryEdit">
            <property name="text">
             <string>Z:\ML\English-Handwritten\Img</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openDirectoryButton">
            <property name="text">
             <string>....</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="openDatasetButton">
            <property name="text">
             <string>Open dataset</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2. Configure processing options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="processingOptionsTab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pictureResizerTab">
         <attribute name="title">
          <string>Picture resizer</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="pictureResizerCheck">
              <property name="text">
               <string>Enable picture resizer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Target image dimensions :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pictureWidthEdit">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>pixels</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pictureHeightEdit">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>pixels</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Current dataset:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="datasetNameLabel">
          <property name="text">
           <string>No dataset loaded</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="text">
         <string>About this program</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtDatasetConverter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openDirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>QtDatasetConverterClass</receiver>
   <slot>SelectDatasetDirectoryClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1237</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>1055</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>QtDatasetConverterClass</receiver>
   <slot>StartClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>1021</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openDatasetButton</sender>
   <signal>clicked()</signal>
   <receiver>QtDatasetConverterClass</receiver>
   <slot>OpenDatasetClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>QtDatasetConverterClass</receiver>
   <slot>AboutClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1103</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectDatasetDirectoryClick()</slot>
  <slot>StartClick()</slot>
  <slot>OpenDatasetClick()</slot>
  <slot>AboutClick()</slot>
 </slots>
</ui>
